components/Icon/Icon.jsx

```diff
@@ -165,6 +166,7 @@ const Icon = ({ name, skin = '', style = {}, size = 'medium', ...props }) => {
     credit_card: CreditCard,
     date_range: DateRange,
     delete: Delete,
+    description_outline: DescriptionOutlinedIcon,
```

Comment:
```suggestion
    description_outlined: DescriptionOutlined,
```

Reply:
What do you think about adjusting the name to "outlined"? This way we can keep the nomenclature in accordance with the name of the icon and the other icons already present.